nuntium/user_section/tests/user_section_views_tests.py

```diff
@@ -169,7 +169,7 @@ def test_writeitinstance_advanced_form_save(self):
     @override_settings(OVERALL_MAX_RECIPIENTS=10)
     def test_max_recipients_cannot_rise_more_than_settings(self):
         '''The max number of recipients in an instance cannot be changed using this form'''
-        url = reverse('writeitinstance_advanced_update', kwargs={'slug': self.writeitinstance.slug})
+        url = reverse('writeitinstance_basic_update', kwargs={'slug': self.writeitinstance.slug})
```

Comment:
some of this tests are likely to break but we can get them fixed as we fix the others as well =)